front-end/src/components/Admin/Grid/Filter/index.js

```diff
@@ -46,9 +48,11 @@ class Filter extends Component {
     return fields;
   };
 
+  applyFilter = () => this.apply_filter.classList.add('filter__apply-btn-active');
```

Comment:
This should be done in react-way. There is no need to use refs for this task.
You need to think of the 'enabled filter' as a state of the Filter component: add boolean property 'filtered' to the component state and insert the right class in render() depending on it.